_maps/aetherwhisp.json

```diff
@@ -12,5 +12,6 @@
 	    "emergency": "emergency_donut"},
 	"mine_disable": 1,
 	"ship_type": "/obj/structure/overmap/nanotrasen/solgov/aetherwhisp/starter",
-	"faction": "solgov"
+	"faction": "solgov",
+	"starmap_path": "config/starmap/starmap.json"
```

Comment:
Is SSmapping actually set up to use this?

Reply:
Yeah it should be on 182 of map_config.dm